SpywareFilter/sections/tracking_servers.txt

```diff
@@ -76,6 +76,10 @@
 ||us.tags.newscgp.com^
 ||smazaz.icu^
 ||mpraven.org^
+||beacon.cdn.qq.com^
+||beacon.qq.com^
+||aegis.qq.com^
```

Comment:
qq => first-party

Reply:
I mean file `tracking_servers_firstparty.txt`

Reply:
it was my oversight.